src/inmanta/env.py

```diff
@@ -101,6 +101,21 @@ def _init_env(self) -> bool:
                 return False
             LOGGER.debug("Created a new virtualenv at %s", self.env_path)
 
+        else:
+            # make sure the venv hosts the same python version as the running process
+            if sys.platform != "win32":
+                # get version as a (major, minor) tuple for the venv and the running process
+                venv_python_version = subprocess.check_output([python_bin, "--version"]).decode("utf-8").strip().split()[1]
```

Comment:
In https://github.com/inmanta/inmanta-core/commit/4e889fe930c811dc5951d3cce5d020dcb45ce6f4 we only invoke `python --version` on windows systems. Here we do it the other way around and no check is done on Linux systems. Why?

Reply:
That's exactly what my assumption was :+1: